doc/source/user_guide/changelog/v2.0.0.rst

```diff
@@ -0,0 +1,82 @@
+What's new in 2.0.0 (16 May 2025)
+---------------------------------
+
+These are the changes in icepyx 2.0.0 See :ref:`release` for a full changelog
+including other versions of icepyx.
+
+.. warning::
+    This is major release and contains a few breaking changes. Full details will be
+    described in the documentation.
+
+    NSIDC (the NASA DAAC that hosts ICESat-2 data) is transitioning their
+    API endpoint from the on-premises EGI system to the cloud-based Harmony services.
+    icepyx has been updated to access these new services; Certain capabilities are temporarily going away
+
+    * Data reformatting: users will only get HDF5 files back from the new API.
```

Comment:
what if the product is gridded (so natively a netcdf)? I assume this is referring to the fact that reformatting is not available?

Reply:
@betolink?

Reply:
so maybe the comment should say "No file reformatting is available for now."

Reply:
And this is in "deprecated" because I had the impression that EGI did some reformatting?

Reply:
How about something like this?

```suggestion
    * Data reformatting: Currently reformatting is not supported for any ICESat-2 products, so users will only get the full HDF5 files back from the new Harmony API.
```

Reply:
I think we should stay with something along the lines of "no format transcoding is available" because "full HDF5" sounds like not even subsetting is an option. What do you think?

Reply:
```suggestion
    * Data reformatting: Currently no format transcoding is available for any ICESat-2 products, so users will only get HDF5 files back from the new Harmony API.
```

Reply:
Ah, we can't push these changes because the `development` branch is protected (and I don't think we have permissions to override it). I guess we'll just leave this as is, and update the changelog in a v2.0.1 release.

Reply:
I've updated this sentence on the GitHub release notes at least - https://github.com/icesat2py/icepyx/releases/tag/v2.0.0. Cheers everyone for making this happen!